integration-test/skeleton/serverless.yml

```diff
@@ -6,17 +6,17 @@ provider:
 
 functions:
   main:
-    handler: shtest.main
+    handler: src/ThirdParty.handler
 
   apigw:
-    handler: shtest.apigw
+    handler: src/ApiGateway.handler
     events:
       - http:
           path: hello/{name}
           method: get
 
   subdir:
-    handler: subdir/subtest.main
+    handler: src/SubDir/SubModule.handler
```

Comment:
The original version supported _multiple Stack projects_. If this is removed, there's no reason to include this.

Reply:
Actually yes, because this is what the runtime dispatcher uses for knowing which handler to use.

Reply:
I can't for the life of me get this to work, I keep getting errors about not being able to find an executable named `handler`.